Forge.Forms/src/Forge.Forms/DynamicExpressions/Resource.cs

```diff
@@ -127,6 +145,78 @@ protected internal static IValueConverter GetValueConverter(IResourceContext con
             throw new InvalidOperationException($"Value converter '{valueConverter}' not found.");
         }
 
+        protected IMultiValueConverter GetMultiValueConverter(IResourceContext context)
+        {
+            return GetMultiValueConverter(context, ValueConverter);
+        }
+
+        protected internal static IMultiValueConverter GetMultiValueConverter(IResourceContext context, string valueConverter)
+        {
+            if (string.IsNullOrEmpty(valueConverter))
+            {
+                return null;
+            }
+
+            object parameter = null;
+            var index = valueConverter.IndexOf(':');
+            if (index > 0)
```

Comment:
Man I regret doing spaghetti parsing. Hopefully in the future we can use organize this in a more structured approach.